qutebrowser/misc/utilcmds.py

```diff
@@ -247,3 +247,33 @@ def log_capacity(capacity: int):
         raise cmdexc.CommandError("Can't set a negative log capacity!")
     else:
         log.ram_handler.change_log_capacity(capacity)
+
+
+@cmdutils.register(debug=True, name='debug-loglevel')
+def log_level(level=""):
+    """Change the log level for console logging.
+
+    Args:
+       level: log level for console log.
+    """
+    acceptable_levels=['critical','error','info','debug','warning','vdebug']
+    if level in acceptable_levels:
+        log.change_loglevel(level)
+    else:
+        raise cmdexc.CommandError("Invalid argument, choose from ('critical', 'error', 'warning', 'info', 'debug' or 'vdebug')!")
```

Comment:
Instead of doing this validation by hand, you can add a `@cmdutils.argument('level', ['critical', ...])` decorator below `@cmdutils.register` instead, see `def navigate` in `browser/commands.py` for an example.

Instead of duplicating the loglevels here, you could simply use `choices=[level.lower() for level in log.LOG_LEVELS]` (see [list comprehensions](https://docs.python.org/3.5/tutorial/datastructures.html#list-comprehensions)).